fedimint-server/src/config/api.rs

```diff
@@ -875,12 +943,17 @@ mod tests {
             hashes.insert(peer.client.get_verify_config_hash().await.unwrap());
         }
         for peer in &followers {
-            peer.client.verify_configs(hashes.clone()).await.unwrap()
+            peer.client.verify_configs(hashes.clone()).await.unwrap();
         }
 
-        for peer in followers {
-            peer.server.stop().expect("server stops");
+        // start consensus
+        for peer in &followers {
+            peer.client.start_consensus().await.unwrap();
+        }
+
+        // shutdown
+        for peer in &followers {
+            peer.task_group.shutdown().await;
         }
```

Comment:
Do we know that any epochs were produced here? Could it be that consensus started but no epochs were produced?

Reply:
My next PR will attempt to handle the upgrade + shutdown and restart from config API again case.